build.cake

```diff
@@ -302,7 +298,7 @@ Task("TestNetStandard20Engine")
     .OnError(exception => { UnreportedErrors.Add(exception.Message); })
     .Does(() =>
     {
-        RunDotnetNUnitLiteTests(NETCORE_ENGINE_TESTS, "netcoreapp2.1");
+        RunDotnetNUnitLiteTests(NETCORE_ENGINE_TESTS, "netcoreapp3.1");
```

Comment:
Ditto

Reply:
Same as above... it's supposed to test the netstandard engine but isn't doing it.

Reply:
Postponing this.